### PR TITLE
Add a utility for selecting a random element from a list

### PR DESCRIPTION
**Pull request by krame505**
# Changes
Added a utility function for selecting a random element from a list.

# Documentation
I added a doc comment for the new function.

### CODE REVIEW

**Review by remexre — COMMENTED** [review]
rip, do we not have an `nth :: (a ::= [a] Int)`? LGTM if not; though maybe worth noting that it's partial on an empty list? (Maybe I'll add `@partial` for doc comments at some point?)

**Review by remexre — APPROVED** [review]

**Comment by krame505** [comment]
Yeah, doesn't look like we have `nth` but I improved the error message and added a comment for when the list is empty.

**Pull request merged**
